CompRep.tex

```diff
@@ -36,9 +36,9 @@ \section{Introduction\label{sec:intro}}
 is necessary to extend PSyclone to fully parse individual Fortran
 statements in the Kernel layer and then, for example, annotate the
 source code with directives such as OpenACC or OpenMP. This work is just beginning,
-however, the micro-benchmark~\cite{lfric-microbenchmarks} suite has been developed so computational
-intensive kernels can be used to explore what kernel optimisation can
-be developed which can then be generated by PSyclone in the full
+however, the micro-benchmark~\cite{lfric-microbenchmarks} suite has been 
+developed so compute intensive kernels can be used to explore what kernel
+optimisation can be developed which can then be generated by PSyclone in the full

```

Comment:
Suggestion: maybe this is clearer?

"used to explore what kernel optimisation can be developed which can then be generated..."
to
"used to explore kernel optimisations which can then be generated..."